monorepo/website/src/components/asides/InfoBox.astro

```diff
@@ -2,8 +2,8 @@
 import MaterialSymbolsInfoOutline from '~icons/material-symbols/info-outline';
 ---
 
-<div class='bg-primary-500 !text-white p-3 my-4 rounded-lg max-w-md mx-auto box text-sm'>
-    <span style={{ fontSize: '1.0rem' }} class='!text-white'>
+<div class='bg-primary-300 p-3 my-4 rounded-lg mx-auto box text-sm'>
```

Comment:
Can you show a screenshot of the changes here? What prompted you to make these changes? Just curious

Reply:
You can see it here:
https://preview-update-docs-group-edit.pathoplexus.org/about/terms-of-use/open-data
https://pathoplexus.org/about/terms-of-use/open-data

I have slightly mixed feelings, but this probably is an improvement